docs/user/reproducibility.rst

```diff
@@ -0,0 +1,177 @@
+.. _reproducibility:
+
+Reproducibility
+===============
+
+First and foremost, we have designed Renku to enable reproducible data
+science. In addition, Renku enables you to share pieces of your analysis
+process with others at any time, and conversely, to reuse the work of others.
+We encourage such cooperation by recording use of both data and code so that
+the lineage and provenance of both can be uniquely determined.
+
+
+In a typical data analysis or modeling project, these types of questions arise
+very frequently:
+
+* How exactly was this model or result obtained? How was the raw data pre-processed?
+* Who is using my data, model, or result and how?
+* What does this new data mean for our last month's report?
+* How can I share my latest analysis with a colleague in the cloud?
+* How can I make sure that our colleagues in another team can reproduce our results exactly?
+
+Answering these questions in a reliable fashion is difficult. Renku provides
+the tools that substantially reduce the effort required on the part of the
+data scientist who wants to keep her work reproducible, reliable and robust.
```

Comment:
--> [...] required on the part of data scientist**s** who wants to keep **their** work reproducible [...]